package.json

```diff
@@ -34,7 +33,6 @@
   },
   "homepage": "https://github.com/SonarSource/SonarTS",
   "peerDependencies": {
-    "tslint": "^5.1.0",
```

Comment:
you must keep this peer dependency, it tells that our tslint-sonarts plugin requires at least tslint@5.1.0